src/pages/Home.svelte

```diff
@@ -1,23 +1,169 @@
 <script lang="ts">
   import DocSidebar from "@Src/components/DocSidebar/DocSidebar.svelte";
+  import MarkdownIt from "markdown-it";
+  import MarkdownItAnchor from "markdown-it-anchor";
+  import { onMount } from "svelte";
+  import Fa from "svelte-fa/src/fa.svelte";
+  import { faArrowUp } from "@fortawesome/free-solid-svg-icons";
+
+  import type { SidebarItem } from "@Src/components/DocSidebar/SidebarItemTypes";
+  import { ElementType } from "@Src/components/DocSidebar/SidebarItemTypes";
+  import Arrow from "../assets/Arrow.svg";
+  import "../styles/home.css";
+
+  export type HtmlTitle = {
+    tokenContent: string;
+    innerText: string;
+  };
+
+  const md = new MarkdownIt({
+    html: true,
+    linkify: true,
+    typographer: true,
+  });
+  md.use(MarkdownItAnchor);
+
+  let finalPage: string;
+  let hideAccessToGraph = false;
+
+  let summary: Array<SidebarItem> = [];
+
+  onMount(async () => {
+    const mdGettingStarted = await GetReadmeContent(
+      window.location.pathname,
+      "src/markdown/GETTING_STARTED.md"
+    );
+    const mdDdd = await GetReadmeContent(window.location.pathname, "src/markdown/DDD.md");
+    if (mdGettingStarted && mdDdd) {
```

Comment:
I think it's useless to loop on it, because the md repository is not supposed to change dynamically, i.e the files are already known. It's easier and take less resources to directly look for a known file.